src/simple-keyring.test.ts

```diff
@@ -135,22 +168,27 @@ describe('simple-keyring', function () {
     it('reliably can decode messages it signs', async function () {
       await keyring.deserialize([privateKey]);
       const localMessage = 'hello there!';
-      const msgHashHex = bufferToHex(keccak256(Buffer.from(localMessage)));
+      const msgHashHex = bufferToHex(
+        Buffer.from(keccak256(Buffer.from(localMessage))),
+      );
 
       await keyring.addAccounts(9);
       const addresses = await keyring.getAccounts();
       const signatures = await Promise.all(
-        addresses.map(async (accountAddress) => {
+        addresses.map(async (accountAddress: Hex) => {
           return await keyring.signMessage(accountAddress, msgHashHex);
         }),
       );
       signatures.forEach((sgn, index) => {
         const accountAddress = addresses[index];
 
+        /* eslint-disable id-length */
         const r = toBuffer(sgn.slice(0, 66));
-        const s = toBuffer(`0x${sgn.slice(66, 130)}`);
-        const v = BigInt(`0x${sgn.slice(130, 132)}`);
+        const s = toBuffer(`0x${sgn.slice(66, 130) as string}`);
```

Comment:
Sorry to keep asking about this, but why the typecast? Can we use some other way to create a string?

Reply:
Hmm, if you write `const s = toBuffer('0x' + sgn.slice(66, 130));`

Lint will complain:
```
187:28  error  Operands of '+' operation with any is possible only with string, number, bigint or any  @typescript-eslint/restrict-plus-operands
87:28  error  Unexpected string concatenation                                                         prefer-template
```

Reply:
Hmm. I just tried removing the typecast locally and am not getting this error. That said, what does it think `sgn.slice(66, 130)` is?

Reply:
Oh here we go, I needed to tell it sgn is a string on this line
`signatures.forEach((sgn: string, index) => {`